src/modules/services_manager_service.rb

```diff
@@ -47,16 +45,14 @@ def read
         load_units
 
         @supported_unit_files = unit_files.select do |_, status|
-          Status::SUPPORTED_STATES.member?(status)
+          status != Status::MASKED # masked services should not been shown at all
```

Comment:
We are supporting now almost all states except the "mask" state. So we will not miss any new state in the future.